cli/src/commands/git/push.rs

```diff
@@ -746,6 +768,63 @@ fn classify_bookmark_update(
     }
 }
 
+/// Creates a bookmark for a single `--named` argument and returns its name
+///
+/// The logic is not identical to that of `jj bookmark create` since we need to
+/// make sure the new bookmark is safe to push.
+fn create_explicitly_named_bookmarks(
+    ui: &mut Ui,
```

Comment:
nit: `ui: &Ui`

Reply:
Thanks! I'm surprised clippy didn't pick up on it.